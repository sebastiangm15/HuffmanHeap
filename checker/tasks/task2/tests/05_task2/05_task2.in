15
10 ST14O
30 ST905
12 STMBE
22 STPC1
5 ST3JM
23 STWHN
24 ST8KK
1 STUKD
13 STDKM
12 STWRA
18 ST4IH
15 STP6G
19 ST3IU
29 STO2K
24 STVAO
10
111
1110
00110
011010
1010010
10000010
000010000
0000111100
11010011110
000000010111
