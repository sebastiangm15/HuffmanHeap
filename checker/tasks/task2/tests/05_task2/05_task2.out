
STWHN 
STVAO 
STO2K 
ST3JM 
ST905 STWRA 
STMBE ST905 
STMBE ST8KK 
STPC1 STVAO STWHN 
STMBE STWRA STO2K 
